pytorch_lightning/loggers/trains.py

```diff
@@ -295,11 +295,9 @@ def log_artifact(
             delete_after_upload=delete_after_upload
         )
 
-    def save(self) -> None:
-        pass
-
     @rank_zero_only
     def finalize(self, status: str = None) -> None:
+        # super().finalize(status)
```

Comment:
Is this intended to be commented? If so, why?

Reply:
I have added this to be the same as other loggers but then it is crashing because the accumulator is missing...
I don't feel like debugging this magic tool, there are already several other things I don't like much... 
It have not been even properly tested yet